indico/modules/rb_new/client/js/modules/roomList/RoomFilterBar.jsx

```diff
@@ -109,6 +125,16 @@ class RoomFilterBarBase extends React.Component {
             <Button.Group size="large">
                 <Button icon="filter" as="div" disabled />
                 <FilterBarController>
+                    <FilterDropdownFactory name="building"
+                                           title={<Translate>Building</Translate>}
+                                           form={({building: selectedBuilding}, setParentField) => (
+                                               <BuildingForm setParentField={setParentField}
+                                                             buildings={Object.keys(buildings)}
```

Comment:
If all we ever use here is the list of building numbers, then let's add a selector for this:

```js
export const getBuildingNames = createSelector(
    getBuildings,
    buildings => Object.keys(buildings),
);
```

Reply:
Actually, I just realized that we don't use the `getBuildings` selector anywhere else and the floor information isn't needed anymore. So we can just simplify `getBuildings` and remove all the now-unnecessary logic from it:

```js
export const getBuildings = createSelector(
    getAllRooms,
    (allRooms) => Array.from(new Set(allRooms.map(room => room.building)))
);
```